src/vanilla/ClientModelExtensions.cs

```diff
@@ -661,6 +567,26 @@ public static void ConstructRequestBodyMapper(TSValue value, ParameterTS request
             }
         }
 
+        public static void ConstructResponseBodyMapper(TSValue value, Response response, Method method)
+        {
+            IModelType responseBodyModelType = response.Body;
+            if (responseBodyModelType is CompositeType)
+            {
+                value.Text($"Mappers.{responseBodyModelType.Name}");
+            }
+            else
+            {
+                ConstructMapper(
+                    value,
+                    responseBodyModelType,
+                    "parsedResponse",
+                    null,
```

Comment:
Please label this and any of the above arguments that aren't self-explanatory based on the values/names

Reply:
Fixed in https://github.com/Azure/autorest.typescript/pull/177/commits/f49ffa6c69fab6da9cdaa26f1b3f60b4bc4f794c.